js/main.js

```diff
@@ -50,6 +51,7 @@
     var apiUrl      = baseAPIsUrl + "2";
     var assetsUrl   = "https://d116d2b4cs97oy.cloudfront.net";
     var aWSS3Assets = "https://s3.amazonaws.com/classroom-assets";
+    var imgixUrl    = "http://www.imgix.net/";
```

Comment:
I don't know if this subdomain will be always available, check against our subdomain, but ask them to whitelist all of imgix.net